### PR TITLE
Doc Changes

### PR DESCRIPTION
**Pull request by Krunal-Thakkar**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 80% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -96,17 +154,21 @@ The release supports resources and data sources mentioned in the Features sectio
...
 
 ### Others
-N/A
-## Enhancements
+
 N/A
 
-## Bug Fixes
+### Enhancements
+
```
Comment (AnikaAgiwal2711) [inline CHANGELOG.md:169]: we should add the Enhancements done here. Right ??

Comment (Krunal-Thakkar) [reply CHANGELOG.md:169]: Added on v1.4.0 section

Comment (AnikaAgiwal2711) [reply CHANGELOG.md:169]: it's already added. my mistake

**Review by AnikaAgiwal2711 — DISMISSED** [review]

**Comment by AnikaAgiwal2711** [comment]
One change which needs to be done here : https://github.com/dell/terraform-provider-powerflex/blob/main/examples/provider/provider.tf 
version will be 1.4.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,87 +7,145 @@ You may obtain a copy of the License at
...
+  
+### Enhancements
+
+* All existing resources and datasources are qualified against PowerFlex v4.5 on Azure.
+* Added support for including SDS in fault set.
+* Added support virtual ip/interfcaes in SDC resource.
+  
+### Bug Fixes
+
+* Fixed the change in value of is_sdc from No to Yes which was previously giving an error: Provider produced inconsistent results after apply.
```
Comment (shenda1) [inline CHANGELOG.md:44]: SDC resource: Fixed the change in the value of is_sdc from No to Yes, which previously resulted in the error: "Provider produced inconsistent results after apply"

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Review by shenda1 — APPROVED** [review]

**Pull request merged**